tests_pycsob/test_api.py

```diff
@@ -19,14 +19,15 @@
 class CsobClientTests(TestCase):
 
     def setUp(self):
+        self.key = open(KEY_PATH).read()
```

Comment:
Asi nejrychlejší způsob, jak to zprovoznit - CsobClient sice po posledních změnách může přijímat i path, ale funkce z utils nemohou....